internal/services/iothub/iothub_resource.go

```diff
@@ -376,6 +376,14 @@ func resourceIotHub() *pluginsdk.Resource {
 						},
 
 						"resource_group_name": commonschema.ResourceGroupNameOptional(),
+
+						// We have to add "Computed: true" since this property would always be set when it isn't specified in the tf config, otherwise it would block the existing users
+						"subscription_id": {
```

Comment:
Dose it make sense to name this property as `endpoint_subscription_id` or another naming that could more intuitively self-explain what itself is in business, otherwise users might be confused what the difference between this newly added subs_id and the default subs_id is?

Reply:
Per the previous comment from Hashicorp, they may suggest to not name it as "endpoint_subscription_id" since the parent property name is "endpoint" and it would be duplicate with "endpoint_subscription_id". I assume that current property schema has indicates that this "subscription_id" is for "endpoint".

Reply:
This is fine as this attribute is already in the `endpoint` block.